app/jobs/invoices/provider_taxes/void_job.rb

```diff
@@ -6,7 +6,7 @@ class VoidJob < ApplicationJob
       queue_as "integrations"
 
       def perform(invoice:)
-        return unless invoice.customer.anrok_customer
+        return unless invoice.customer.tax_customer
```

Comment:
could we also cover this into the job spec? right it has two scenarios anrok customer, not anrok customer. At least update the description of the spec to let the reader know it is not only anrok customer but tax customer...

Reply:
added one more test case 👍